lib/smart_answer_flows/energy-grants-calculator.rb

```diff
@@ -470,6 +470,9 @@ def define
         precalculate :flat_type do
           flat_type
         end
+        precalculate :under_green_deal do
```

Comment:
@ikennaokpala: It's probably worth adding a sentence or two to explain this in the [storing data docs][1]. I've added a [Trello card][2] to capture this.

[1]: https://github.com/alphagov/smart-answers/blob/master/doc/storing-data.md
[2]: https://trello.com/c/Ldn6KJeE